### DISCUSSION TITLE
Welcome to WanderLust Discussions!

### DISCUSSION THREAD

**Discussion by SabaNadeemUOS** [SabaNadeemUOS/WanderLust#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SabaNadeemUOS** [comment]
Here is the layout of the page.It is a two column layout
![image](https://user-images.githubusercontent.com/113013957/200188659-8c2feeb7-5740-4755-99c2-c57e7cb667e7.png)

**Comment by SabaNadeemUOS** [comment]
I also added a small plane animation

https://user-images.githubusercontent.com/113013957/200188963-be5c8aa1-c1fd-40f0-b840-643fe472dd5c.mp4

**Reply by sarahalhammadi** [reply]
i like the idea of the plane animation

**Comment by Mouza-Ayman-Alsharhan** [comment]
hello guys... i did push some changes to our repository.
i did create CSS file and added to it rules for styling the navigation bar.
have a look at it, and tell me everything if there is anything that i can fix.
Saba, please do a pull request for my branch.
![Screenshot 2022-11-08 182030](https://user-images.githubusercontent.com/113240180/200589011-bbf57db5-c080-40d1-845e-dd7c3b74511a.png)

**Reply by SabaNadeemUOS** [reply]
I think the navigation looks perfect!
I will create the pull request now

**Comment by shaikhaalz** [comment]
why we create classes for nav ? , why we don't use ul direct in css file ?

**Reply by Mouza-Ayman-Alsharhan** [reply]
that's becuse we want to implement these CSS rules for navigation only, and we might have other ul elements in the html.

**Reply by shaikhaalz** [reply]
oh yeah you are right, i didn't think about that

**Comment by shaikhaalz** [comment]
what do you think to add something in background like a unique color better than white

**Reply by SabaNadeemUOS** [reply]
We could add a background picture but then we would have to find suitable colors for the text

**Reply by shaikhaalz** [reply]
it is okay we will check for that after we finish the main design

**Comment by SabaNadeemUOS** [comment]
Why do we have a google map on the home page?I think we can replace it with an image
![image](https://user-images.githubusercontent.com/113013957/201135685-442b2dc3-1ef5-4916-8a8b-74963aaa9cd7.png)

**Reply by shaikhaalz** [reply]
i agree, it will be elegant more

**Reply by Mouza-Ayman-Alsharhan** [reply]
well, I thought since it is a website about traveling, adding a map for the world and showing all countries would be nice.
of course, if you guys didn't like it we can replace it with an image.

**Reply by shaikhaalz** [reply]
or we can locate the location as a hyperlink on the map image, which i mean when they press on the photo it will transfer them to google map direct !

**Reply by SabaNadeemUOS** [reply]
Maybe we can add a map that has markers of the places we have been too? Or maybe it is possible to create a map animation showing the places we have talked about
But that seems hard

**Reply by shaikhaalz** [reply]
yes it will be hard

**Comment by SabaNadeemUOS** [comment]
Guys is this font okay for the headings?
![image](https://user-images.githubusercontent.com/113013957/201170637-62460102-f0e5-444f-ad60-75f45b3b00ea.png)

**Reply by shaikhaalz** [reply]
ig let's make it a unique font such as ( Garamond or Times New Roman or Georgia ).

**Reply by SabaNadeemUOS** [reply]
this is an imported font i was trying to add something new to the css but okay i will try to find some other font

**Comment by SabaNadeemUOS** [comment]
This looks okay?It looks empty....
![image](https://user-images.githubusercontent.com/113013957/201460378-9724bce1-d655-4859-ae49-1754c1f1ad95.png)

**Reply by shaikhaalz** [reply]
What do you think about this styles;
![73AA7918-C7B9-4D01-8730-18ECD145BB0F](https://user-images.githubusercontent.com/113843277/201465561-4130810a-f602-4459-8f89-ba1ce28d7002.jpeg)
![36FE0C04-51AF-4E24-A5C1-9A41B9EE944A](https://user-images.githubusercontent.com/113843277/201465652-af0c3ddf-1a21-466c-b0cf-cd9fff1d9776.jpeg)
![FF8C9495-6433-4D09-8BF0-B43611046070](https://user-images.githubusercontent.com/113843277/201465655-884ce502-e45e-4bcf-9770-b3235ddccd97.jpeg)
Take some idea from them

**Reply by SabaNadeemUOS** [reply]
i was trying something like this but the problem is i can't align them properly
I am facing issues with that

**Reply by SabaNadeemUOS** [reply]
whatever i write gets pushed down by the list
i don't know how to keep them in the same line
I tried using inline and inline-block but it didn't work
